waltz-ng/client/common/tally-utils.js

```diff
@@ -34,4 +34,31 @@ export function tallyBy(data = [], groupingSelector = 'key') {
         .countBy(groupingSelector)
         .map((v, k) => ({ key: k, count: v }))
         .value()
-}
\ No newline at end of file
+}
+
+
+/**
+ * Given a node with a @countKey, adds total and child counts for the node
```

Comment:
This performs a sum on the given node only, it is involved from org-units/list-view.js and the data-type-tree.js.  

Another lift and shift, renamed to: buildPropertySummer